controllers/attractor/lb-fe.go

```diff
@@ -69,15 +69,14 @@ func (l *LoadBalancer) getLbEnvVars(allEnv []corev1.EnvVar) []corev1.EnvVar {
 			Name:  "NSM_NSP_SERVICE",
 			Value: common.NSPServiceWithPort(l.trench),
 		},
-		{
-			Name:  "NSM_NAME",
-			Value: lbFeDeploymentName,
-		},
 	}
 
 	for _, e := range allEnv {
 		// append all hard coded envVars
-		if e.Name == "SPIFFE_ENDPOINT_SOCKET" {
+		if e.Name == "SPIFFE_ENDPOINT_SOCKET" ||
+			e.Name == "NSM_NAME" ||
+			e.Name == "NSM_MAX_TOKEN_LIFETIME" ||
+			e.Name == "NSM_LOG_LEVEL" {
```

Comment:
I don't see these xxx_LOG_LEVEL added in the yaml files. Is it intended to use the default values?

Reply:
Yes, the default for these *LOG_LEVEL env values is controlled through the respective go codes.
I just wanted to add support to the code for including them from the templates if present. But if you think it'd be better to add them explicitly to the templates, then I can do that.